sockeye/arguments.py

```diff
@@ -854,27 +845,21 @@ def add_training_args(params):
                               default=C.LR_SCHEDULER_PLATEAU_REDUCE,
                               choices=C.LR_SCHEDULERS,
                               help='Learning rate scheduler type. Default: %(default)s.')
+    train_params.add_argument('--learning-rate-t-scale',
+                              type=float,
+                              default=1.0,
+                              help="Step number is multiplied by this value when determining learning rate for the "
+                                   "current step. Default: %(default)s.")
     train_params.add_argument('--learning-rate-reduce-factor',
                               type=float,
-                              default=0.7,
+                              default=0.9,
```

Comment:
Do we want to update the default arguments here? Users might have sockeye.train scripts with this parameter not specified and we don't want to change the overall behavior too much.

Reply:
This could be considered part of the training update and general 2.0 release.  Since we're already not going to be backward compatible, we have a chance to update our default training recipe to the best current settings.

We can also give users some sample training commands for 1 GPU, multiple GPUs, and distributed training in the updated readme.